FV3/atmos_cubed_sphere/tools/coarse_grained_restart_files.F90

```diff
@@ -272,8 +273,13 @@ end subroutine register_fv_land_coarse
   subroutine coarse_grain_restart_data(Atm)
     type(fv_atmos_type), intent(inout) :: Atm
 
+    character(len=256) :: error_message
+
     if (trim(Atm%coarse_graining%strategy) .eq. MODEL_LEVEL) then
        call coarse_grain_restart_data_on_model_levels(Atm)
+    else
+       write(error_message, *) 'Currently only model_level coarse-graining is supported for restart files.'
```

Comment:
beautiful coherent error message :)

Reply:
to confirm though: it's not possible to coarse-grain diagnostics on pressure and coarse-grain restarts on model levels in the same run? I think that's okay, just wanted to check.

Reply:
That's correct -- currently you cannot mix model-level and pressure-level coarse-graining in the same run.  We could introduce more complexity in the options, e.g. offer separate controls for restart files and diagnostics, but my thinking for now is where do we draw the line?  In principle one could ask for control over the method on a per-variable basis, which would require a significant change to the design (i.e. we wouldn't want to control that through namelist options).

Reply:
Got it. Yeah I agree we should keep it as is.